src/Features/CSharp/Portable/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -147,7 +148,7 @@ internal static class AddImportDiagnosticIds
     }
 
     [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AddImport), Shared]
-    internal class CSharpAddImportCodeFixProvider : AbstractAddImportCodeFixProvider<SimpleNameSyntax>
+    internal class CSharpAddImportCodeFixProvider : AbstractAddImportCodeFixProvider
```

Comment:
Note: i intentionally spit the types and kept them in the same file to make the PR easier to read.  A following PR will move them into their own files.